schema/image/resized.js

```diff
@@ -17,7 +17,7 @@ export const resizedImageUrl = (image, options) => {
   const desired = _.pick(opts, 'width', 'height');
   const factor = _.min(_.map(desired, (value, attr) => {
     return value / image[`original_${attr}`];
-  }));
+  })) || Infinity;
```

Comment:
Seemed like maybe `_.min` had a different return value in lodash 4—but I can better investigate that, just dropped this in b/c it made the test happy. I dunno if it has unforeseen repercussions.